Encrypted Slack bridges (#493)
matrix-org/matrix-appservice-slack: docs/bridge-encryption.md

Bridge Encryption
=================

> The Slack bridge supports E2EE rooms, using [pantalaimon](https://github.com/matrix-org/pantalaimon).

Reviewer: First mention of an abbreviation.
```suggestion
The Slack bridge supports end-to-end encryption (E2EE) rooms, using [pantalaimon](https://github.com/matrix-org/pantalaimon).
```

This means the bridge will encrypt messages when sending to E2EE rooms, and

> will decrypt them to be sent to Slack. New DMs and private channels will also

Reviewer: ```suggestion
will decrypt them to be sent to Slack. New direct messages (DMs) and private channels will also
```

be encrypted by default when bridged.

> It goes without saying that messages going to and from Slack will not be encrypted,
> so only the Matrix side of the conversation will be secure. This may still be advantagous

Reviewer: Messages to and from Slack are encrypted by HTTPS and further secured by Slack's IT team.
As far as I understood the main benefit is not having to trust (federated) homeservers with company-internal Slack channels.

```suggestion
The end-to-end encryption is established between Matrix clients and the bridge. Messages going to and from Slack will be encrypted by HTTPS but won't use E2EE.
Bridge encryption secures conversation contents on all Matrix homeservers and clients it is sent to. Also, this may be advantageous
```

Author: That's much better. Thank you!
